applications/fishing-map/src/features/dataviews/dataviews.utils.ts

```diff
@@ -140,3 +142,8 @@ export const getContextDataviewInstance = (
   }
   return contextDataviewInstance
 }
+
+export const dataviewWithPrivateDatasets = (dataview: UrlDataviewInstance) => {
+  const datasets = dataview.datasets || []
+  return datasets.some((d) => d.id.includes(PRIVATE_SUFIX))
```

Comment:
Sorry if already asked/obvious: why can't we use a `private` flag on datasets and workspaces?

If not possible, could we at least try to centralize the logic here with https://github.com/GlobalFishingWatch/frontend/pull/815/files#diff-7f372e623785bb5176dfba7113a764c9456a03c8c3707f60c87567f8d5b5258dR26, and also in `user.selectors`

Reply:
Merged in https://github.com/GlobalFishingWatch/frontend/pull/815/commits/9fb6972a546bdbc5470528b475441c1f5aba9bab